### PR TITLE
Create Elevator Pitch '24

### PR DESCRIPTION
**Pull request by Anidion**

### CODE REVIEW

File: src/events/2024-2025/demos/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<!-- This year's showcase is scheduled for **April 27th, 2024**.
+To prepare for this, we want each team to demo and pitch their project to
+the club before the showcase. This will help you get experience and valuable
+feedback from fellow project teams, execs, and our partners in industry.-->
+
+## Dates to Remember
+
+| Date             |           Event |           Audience | Location |
+| ---------------- | --------------: | -----------------: | -------: |
+| 21 November 2024 | Elevator Pitche |         Tech Start |   ENG 03 |
```
Comment (ParsaKargari) [inline src/events/2024-2025/demos/README.md]:
```suggestion
| 21 November 2024 | Elevator Pitch |         Tech Start |   ENGG 03 |
```

File: src/events/2024-2025/demos/elevator-pitch/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Quick Info
+
+### Objectives
+
+Let everyone know what your product idea is, present progress so far and next steps.
+
+### Start Time
+
+6pm sharp
```
Comment (ParsaKargari) [inline src/events/2024-2025/demos/elevator-pitch/README.md]:
```suggestion
6 PM sharp
```

**Review by ParsaKargari — APPROVED** [review]
Small changes/typos, everything looks great

File: src/events/2024-2025/demos/README.md (hunk 1 above)
Comment (Anidion) [reply src/events/2024-2025/demos/README.md]: `ENG` is the right name for the building. Fixed the Pitche typo 

image

**Pull request merged**